src/target/move.c

```diff
@@ -49,16 +49,16 @@ static void entered_next_cell(void)
  * This functions assumes the current speed is the target speed and takes into
  * account the configured linear deceleration.
  */
```

Comment:
include @param, @return

Reply:
Reviews bellow the line you are reviewing! :stuck_out_tongue_winking_eye:

Reply:
300a8c4a10eacf8547df3cbbf3186a57ee088e05

Reply:
ok